cmake/cmakepp_lang/algorithm/equal.cmake

```diff
@@ -7,35 +7,36 @@ include(cmakepp_lang/types/implicitly_convertible)
 include(cmakepp_lang/utilities/compare_lists)
 include(cmakepp_lang/utilities/return)
 
-#[[[ Compares two values for equivalency.
+#[[[
+# Compares two values for equivalency.
 #
 # This function can be used to compare any two CMakePP objects for equality. For
 # native CMake types, like booleans and integers, equality is defined as being
 # the same string. ``cpp_equal`` will also compare objects like CMakePP maps or
 # instances of user-defined classes to ensure they have the same state. If two
 # objects have different types they are defined to be unequal.
 #
-# :param _e_result: The name to use for variable holding the result.
-# :type _e_result: desc
-# :param _e_lhs: One of the two values involved in the comparison.
-# :type _e_lhs: str
-# :param _e_rhs: The other value involved in the comparison
-# :type _e_rhs: str
+# :param result: The name to use for variable holding the result.
+# :type result: desc
+# :param lhs: One of the two values involved in the comparison.
+# :type lhs: str
+# :param rhs: The other value involved in the comparison
+# :type rhs: str
 # :returns: ``_e_result`` will be set to ``TRUE`` if ``_e_lhs`` compares equal
```

Comment:
Missed removal of `_e_` prefix in the returns field. Looks like this same thing was missed in all of the return documentation

Reply:
No worries, thanks for pointing this out! I just did a quick find and replace for `:param _[a-zA-Z]+_` and `:type _[a-zA-Z]+_` before submitting, so it makes sense that I missed all of those. I'll have to go back in and fix that.